src/inline_tests.ml

```diff
@@ -95,7 +95,7 @@ module Backend = struct
          ; field "flags" Ordered_set_lang.Unexpanded.sexp_of_t t.info.flags
          ; field_o "generate_runner" Action.Unexpanded.sexp_of_t
              t.info.generate_runner
-         ; field_o "extends" (list f) (Option.map t.extends ~f:Result.ok_exn)
+         ; field "extends" (list f) (Result.ok_exn t.extends) ~default:[]
```

Comment:
The change to `field` makes sure the compatibility with old backends, right?

Reply:
Yes, it should be the same S-expression representation